src/coreclr/gc/gc.cpp

```diff
@@ -13662,8 +13662,8 @@ gc_heap::init_semi_shared()
 
     memset (full_gc_counts, 0, sizeof (full_gc_counts));
```

Comment:
```suggestion
    full_gc_counts = {};
```

Reply:
I can't do that since `full_gc_counts` is an already instantiated array. `memset` is the preferred way to do this unfortunately.